templates/_navbar_with_login.html.erb

```diff
@@ -38,11 +38,6 @@
                   <i class="fa fa-home"></i> Flats
                 <% end %>
               </li>
-              <li>
```

Comment:
@krokrob this is the template WITH devise login. 
So it needs the `destroy_path` link